tox.ini

```diff
@@ -12,8 +12,10 @@ pytest_cov_args = --cov --cov-report html --cov-report term --cov-report xml
 
 [testenv]
 description = Run tests with {basepython}
```

Comment:
```suggestion
description = Run tests with {basepython}
```
```suggestion
description =
    Run tests with {basepython}
    !unit-devel,!integration-devel: DO NOT USE the `devel` factor without `unit` or `integration`
```

Reply:
Is there documentation on how factors work within a `description`? I couldn't find any and find this confusing. I would expect that warning to NOT show up with `unit-devel` or `integration-devel` just based on what I know about factors, but they do.